truss/test_data/context_builder_image_test/Dockerfile

```diff
@@ -4,6 +4,6 @@ COPY test.py test.py
 
 # This seems to be present on github actions execution environment and interferes.
 # So, get rid of it.
-RUN rm -rf /.venv
+# RUN rm -rf /.venv
```

Comment:
@pankajroark are you opposed to me dropping this  – I think this was the reason for the failures on the integration tests I was observing. Want to make sure I'm not missing anything